pkg/multiTenancyPlugins/executor.go

```diff
@@ -28,10 +28,11 @@ func (*Executor) Handle(cluster cluster.Cluster, swarmHandler http.Handler) http
 	}
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log.Debug(r)
-		err := startHandler(utils.ParseCommand(r), cluster, w, r, swarmHandler)
-		if err != nil {
-			log.Error(err)
-			http.Error(w, err.Error(), http.StatusBadRequest)
+		var errInfo utils.ErrorInfo
+		errInfo = startHandler(utils.ParseCommand(r), cluster, w, r, swarmHandler)
+		if errInfo.Err != nil {
+			log.Error(errInfo.Err)
+			http.Error(w, errInfo.Err.Error(), errInfo.Status)
```

Comment:
On many cases you didn't setup the status , so maybe test if status is set and if not use the general debrequest error status